sdk/monitor/Azure.Monitor.Ingestion/tests/MonitorIngestionLiveTest.cs

```diff
@@ -175,5 +175,33 @@ public void ConcurrencySingleThread()
             Assert.AreEqual(UploadLogsStatus.Success, response.Value.Status);
             Assert.IsEmpty(response.Value.Errors);
         }
+
+        [Test]
+        public async Task ValidInputFromObjectAsJsonNoBatchingAsync()
```

Comment:
Do we also need a test that passes in some other content-encoding now that we are exposing that parameter?